driver display display_st7789v.c build error

**Describe the bug**
display_st7789v.c misc build error

**To Reproduce**
Steps to reproduce the behavior:
```
[237/336] Building C object zephyr/drivers/display/CMakeFiles/drivers__display.dir/display_st7789v.c.obj
FAILED: zephyr/drivers/display/CMakeFiles/drivers__display.dir/display_st7789v.c.obj
C:\gnu_arm_embedded\bin\arm-none-eabi-gcc.exe -DBUILD_VERSION=v2.7.0-rc2-121-g611284021cef -DKERNEL -DLV_CONF_INCLUDE_SIMPLE=1 -DNRF52832_XXAA -D_FORTIFY_SOURCE=2 -D__LINUX_ERRNO_EXTENSIONS__ -D__PROGRAM_START -D__ZEPHYR_SUPERVISOR__ -D__ZEPHYR__=1 -I../../../../include -Izephyr/include/generated -I../../../../soc/arm/nordic_nrf/nrf52 -I../../../../lib/libc/newlib/include -I../../../../subsys/bluetooth -IC:/Users/GW00221994/zephyrproject/modules/hal/cmsis/CMSIS/Core/Include -IC:/Users/GW00221994/zephyrproject/modules/hal/nordic/nrfx -IC:/Users/GW00221994/zephyrproject/modules/hal/nordic/nrfx/drivers/include -IC:/Users/GW00221994/zephyrproject/modules/hal/nordic/nrfx/mdk -I../../../../modules/hal_nordic/nrfx/. -IC:/Users/GW00221994/zephyrproject/modules/debug/segger/SEGGER -IC:/Users/GW00221994/zephyrproject/modules/debug/segger/Config -I../../../../modules/segger/. -IC:/Users/GW00221994/zephyrproject/modules/crypto/tinycrypt/lib/include -I../drivers/bma421/zephyr/. -I../drivers/bma421/zephyr/bma421 -I../drivers/cst816s/zephyr/. -Os -imacros C:/Users/GW00221994/zephyrproject/zephyr/samples/pinetime/app/build/zephyr/include/generated/autoconf.h -ffreestanding -fno-common -g -gdwarf-4 -fdiagnostics-color=always -mcpu=cortex-m4 -mthumb -mabi=aapcs -mfp16-format=ieee -imacros C:/Users/GW00221994/zephyrproject/zephyr/include/toolchain/zephyr_stdint.h -Wall -Wformat -Wformat-security -Wno-format-zero-length -Wno-main -Wno-pointer-sign -Wpointer-arith -Wexpansion-to-defined -Wno-address-of-packed-member -Wno-unused-but-set-variable -Werror=implicit-int -fno-asynchronous-unwind-tables -fno-pie -fno-pic -fno-strict-overflow -fno-reorder-functions -fno-defer-pop -fmacro-prefix-map=C:/Users/GW00221994/zephyrproject/zephyr/samples/pinetime/app=CMAKE_SOURCE_DIR -fmacro-prefix-map=C:/Users/GW00221994/zephyrproject/zephyr=ZEPHYR_BASE -fmacro-prefix-map=C:/Users/GW00221994/zephyrproject=WEST_TOPDIR -ffunction-sections -fdata-sections -specs=nano.specs -std=c99 -MD -MT zephyr/drivers/display/CMakeFiles/drivers__display.dir/display_st7789v.c.obj -MF zephyr\drivers\display\CMakeFiles\drivers__display.dir\display_st7789v.c.obj.d -o zephyr/drivers/display/CMakeFiles/drivers__display.dir/display_st7789v.c.obj -c C:/Users/GW00221994/zephyrproject/zephyr/drivers/display/display_st7789v.c
C:/Users/GW00221994/zephyrproject/zephyr/drivers/display/display_st7789v.c: In function 'st7789v_pm_control':
C:/Users/GW00221994/zephyrproject/zephyr/drivers/display/display_st7789v.c:410:7: error: void value not ignored as it ought to be
  410 |   ret = st7789v_transmit(data, ST7789V_CMD_SLEEP_IN, NULL, 0);


```

--- a/drivers/display/display_st7789v.c
+++ b/drivers/display/display_st7789v.c
@@ -407,7 +407,7 @@ static int st7789v_pm_control(const struct device *dev,
 		st7789v_exit_sleep(data);
 		break;
 	case PM_DEVICE_ACTION_SUSPEND:
-		ret = st7789v_transmit(data, ST7789V_CMD_SLEEP_IN, NULL, 0);
+		st7789v_transmit(data, ST7789V_CMD_SLEEP_IN, NULL, 0);
 		break;
 	default:
 		ret = -ENOTSUP;

